config-sample.php

```diff
@@ -0,0 +1,14 @@
+<?php
```

Comment:
I prefer to add something like following comments at the beginning part. Feel free to revise as you like.
```
/**
 * Basic configuration of CAPUBBS.
 *
 * This file contains the following configuration: MySQL settings
 *
 * You need to copy this file and rename it to 'config.php' to make it work.
 * You also need to change the corresponding setting to its actual value. For
 * more information, please look at the comments before each item.
 */
```